pkgs/development/libraries/science/math/ecos/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "ecos-${version}";
+  version = "2.0.6";
+
+  src = fetchFromGitHub {
+    owner = "embotech";
+    repo = "ecos";
+    rev = "v${version}";
+    sha256 = "11v958j66wq30gxpjpkgl7n3rvla845lygz8fl39pgf1vk9sdyc7";
+  };
+
+  doCheck = true;
```

Comment:
I think this is true by default

Reply:
I thought so too, but it's definitely not running the `checkPhase` if I omit this and run `nix-build . -A ecos`.  It's definitely defaults to true in the buildPythonPackage helper, but appears not to be true for the general stdenv?

Reply:
Ah yeah you're right, it's false by default